### ISSUE TITLE
CuPy: Support CUDA 12

### ISSUE THREAD

**Issue by leofang** [conda-forge/cupy-feedstock#139]
Creating an issue so that I won't forget...
- The main discussion issue: conda-forge/cudatoolkit-feedstock#62
- @jaimergp's attempt: conda-forge/openmm-feedstock#60

Let's do this and get prepared for the upcoming GPU CI (https://github.com/conda-forge/conda-forge.github.io/issues/1272).

**Comment by leofang** [comment]
WIP in #143.

**Comment by leofang** [comment]
Blocked by https://github.com/conda-forge/staged-recipes/issues/21382.

**Comment by jakirkham** [comment]
Are there any other changes needed in CuPy itself? If not, what release of CuPy includes CUDA 12 support

**Comment by jakirkham** [comment]
Thanks Leo! 🙏

On a separate note, would it be ok if we retitled this issue "CuPy: Support CUDA 12"? Alternatively could file that as a new issue. For context, am trying to name these in a descriptive way when referencing in other places

**Comment by leofang** [comment]
Created a new branch [`cuda-11`](https://github.com/conda-forge/cupy-feedstock/tree/cuda-11) to prepare for the work.
